### PR TITLE
 🐛 Add missing return statement in the webhook admissions func

### PR DESCRIPTION
**Pull request by lleshchi**
Closes: https://github.com/kubernetes-sigs/controller-runtime/issues/2429

### CODE REVIEW

**Comment by FillZpp** [comment]
/ok-to-test

Please add an unit test for this.

File: pkg/webhook/admission/decode.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ func (d *Decoder) DecodeRaw(rawObj runtime.RawExtension, into runtime.Object) er
 			return err
 		}
 		unstructuredInto.SetUnstructuredContent(object)
+		return nil
```
Comment (alvaroaleman) [inline pkg/webhook/admission/decode.go:74]: Could you add a test for the underlying bug, please?

**Comment by sbueringer** [comment]
@lleshchi Do you have some time to add a test for the underlying bug?
Would be nice to get this into the upcoming v0.16 release that we want to cut soon'ish :)

**Comment by lleshchi** [comment]
@sbueringer yes, will add one in today

**Comment by sbueringer** [comment]
Thx!

/lgtm
/approve

**Comment by psalaberria002** [comment]
Would it be possible to release this in 0.15.2? If not, when is 0.16 expected to be out?

**Comment by sbueringer** [comment]
> Would it be possible to release this in 0.15.2? If not, when is 0.16 expected to be out?

I can't promise a specific data, but I assume we're talking about a few days

**Comment by sbueringer** [comment]
I'll cherry-pick independent of that

/cherry-pick release-0.15

**Pull request merged**